mlflow/applications.tf

```diff
@@ -0,0 +1,29 @@
+module "minio" {
+  app_name   = "mlflow-minio"
+  source     = "git::https://github.com/canonical/minio-operator//terraform?ref=track/ckf-1.9"
+  model_name = juju_model.kubeflow[0].name
+  storage_directives = {
+    minio-data = var.minio_size
+  }
+}
+
+module "mlflow_server" {
+  source     = "git::https://github.com/canonical/mlflow-operator//terraform?ref=track/2.15"
+  model_name = juju_model.kubeflow[0].name
+}
+
+module "mlflow_mysql" {
+  # TODO: Update mlflow-mysql once https://github.com/canonical/terraform-modules/pull/20 is merged
+  source              = "git::https://github.com/paulomach/terraform-modules//modules/k8s/mysql-ha-simple?ref=feature/simple-ha-module"
+  juju_model_name     = var.create_model ? juju_model.kubeflow[0].name : local.model_name
+  mysql_app_name      = "mlflow-mysql"
+  mysql_charm_channel = "8.0/stable"
+  mysql_charm_units   = 1
+  # Constrain is a required workaround due to https://github.com/juju/terraform-provider-juju/issues/344
+  mysql_charm_constraints = "arch=amd64"
+  # The following config is equivalent to "constraints: mem=2G"
+  mysql_charm_config = {
+    profile-limit-memory = "2048"
+  }
+  mysql_storage_size = var.mlflow_mysql_size
+}
```

Comment:
I think `model_name = juju_model.kubeflow[0].name` will not work when `create_model` is set to `false`. What we 'd need to do is sth like:
```
  model_name = var.create_model ? juju_model.kubeflow[0].name : var.model_name
```